.github/workflows/required_workflows/plugins/cd.yml

```diff
@@ -0,0 +1,17 @@
+name: cd
+on:
+  workflow_run:
+    workflows: ["ci"]
+    types:
+      - completed
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
```

Comment:
Should this work for RCs? See https://github.com/mattermost/mattermost-plugin-github/pull/625#discussion_r1091723453